cmd/juju/caas/add.go

```diff
@@ -54,7 +70,8 @@ can contain definitions for different k8s clusters, use --cluster-name to pick
 which one to use.
 
 Examples:
-    juju add-k8s myk8scloud
+	juju add-k8s myk8scloud
+	juju add-k8s myk8scloud --region <cloudType/region>
```

Comment:
should add some text to the usageAddCAASDetails above to explain why region is needed, ie when running add-k8s on jaas and the cloud/region cannot be detected automatically